src/transformers/models/iris/modeling_iris.py

```diff
@@ -0,0 +1,1846 @@
+        ...         rewards_tok.append(reward)
+        ...         dones_tok.append(done)
+        ...         observations_tok.append(obs.tolist())
+        ...     observation_tok_batch.append(observations_tok)
+        ...     actions_tok_batch.apped(actions_tok)
+        ...     rewards_tok_batch.append(rewards_tok)
+        ...     dones_tok_batch.append(dones_tok)
+
+        ...     for t in sequence_length_world_model:
+        ...         act = torch.randint(0,4,(batch_size,1)).cpu().numpy() #use rl_agent.act with obs from env.reset in training
+        ...         obs, reward, done, _ = env.step(act)
+        ...         obs = torch.FloatTensor(obs).div(255).permute(0,3,1,2).cpu().numpy()
+        ...         actions_wm.append(act.tolist())
+        ...         rewards_wm.append(reward)
+        ...         dones_wm.append(done)
+        ...         observations_wm.append(obs.tolist())
+        ...     observation_wm_batch.append(observations_wm)
+        ...     actions_wm_batch.append(actions_wm)
+        ...     rewards_wm_batch.append(rewards_wm)
+        ...     dones_wm_batch.append(dones_wm)
+
+        ...     for t in sequence_length_actor_critic:
+        ...         act = torch.randint(0,4,(batch_size,1)).cpu().numpy() #use rl_agent.act with obs from env.reset in training
+        ...         obs, reward, done, _ = env.step(act)
+        ...         obs = torch.FloatTensor(obs).div(255).permute(0,3,1,2).cpu().numpy()
+        ...         actions_ac.append(act.tolist())
+        ...         rewards_ac.append(reward)
+        ...         dones_ac.append(done)
+        ...         observations_ac.append(obs.tolist())
+        ...     observation_ac_batch.append(observations_ac)
+        ...     actions_ac_batch.append(dones_ac)
+        ...     rewards_ac_batch.append(rewards_ac)
+        ...     dones_ac_batch.append(actions_ac)
+
+        >>> observation_tok_batch = torch.tensor(observation_tok_batch).float().to(device)
+        >>> actions_tok_batch = torch.tensor(actions_tok_batch).long().to(device)
+        >>> rewards_tok_batch = torch.tensor(rewards_tok_batch).float().to(device)
+        >>> dones_tok_batch = torch.tensor(dones_tok_batch).long().to(device)
+
+        >>> observation_wm_batch = torch.tensor(observation_wm_batch).float().to(device)
+        >>> actions_wm_batch = torch.tensor(actions_wm_batch).long().to(device)
+        >>> rewards_wm_batch = torch.tensor(rewards_wm_batch).float().to(device)
+        >>> dones_wm_batch = torch.tensor(dones_wm_batch).long().to(device)
+
+        >>> observation_ac_batch = torch.tensor(observation_ac_batch).float().to(device)
+        >>> actions_ac_batch = torch.tensor(actions_ac_batch).long().to(device)
+        >>> rewards_ac_batch = torch.tensor(rewards_ac_batch).float().to(device)
+        >>> dones_ac_batch = torch.tensor(dones_ac_batch).long().to(device)
+
+        >>> mask_padding_tok = torch.ones(batch_size,sequence_length_discrete_autoencoder).bool().to(device)
```

Comment:
pep8 formatting plaease (applies throughout)

```suggestion
        >>> mask_padding_tok = torch.ones(batch_size, sequence_length_discrete_autoencoder).bool().to(device)
```